engine/admin/logging/logentries.md

```diff
@@ -0,0 +1,47 @@
+<!--[metadata]>
++++
+aliases = ["/engine/reference/logging/logentries/"]
+title = "Logentries logging driver"
+description = "Describes how to use the logentries logging driver."
+keywords = ["logentries, docker, logging, driver"]
+[menu.main]
+parent = "smn_logging"
++++
+<![end-metadata]-->
+
+# Logentries logging driver
+
+The `logentries` logging driver sends container logs to the
+[Logentries](https://logentries.com/) server.
+
+## Usage
+
+Some options are supported by specifying `--log-opt` as many times as needed:
+
+ - `logentries-token`: specify the ogentries log set token
+
+Configure the default logging driver by passing the
+`--log-driver` option to the Docker daemon:
+
+    dockerd --log-driver=logentries
```

Comment:
I noticed this was wrong in the document you copied it from, but can you update your code examples to include a `$` prompt? You may want to use GitHub flavored markdown fences instead of identing, and provide a `bash` code-hint to assist in code-highlighting;

    ```bash
    $ dockerd --log-driver=logentries
    ```